sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/noop/NoopDataSource.scala

```diff
@@ -64,3 +72,16 @@ private[noop] object NoopWriter extends DataWriter[InternalRow] {
   override def abort(): Unit = {}
 }
 
+private[noop] object NoopStreamingWriteSupport extends StreamingWriteSupport {
+  override def createStreamingWriterFactory(): StreamingDataWriterFactory =
+    NoopStreamingDataWriterFactory
+  override def commit(epochId: Long, messages: Array[WriterCommitMessage]): Unit = {}
+  override def abort(epochId: Long, messages: Array[WriterCommitMessage]): Unit = {}
+}
+
+private[noop] object NoopStreamingDataWriterFactory extends StreamingDataWriterFactory {
+  override def createWriter(
+      partitionId: Int,
+      taskId: Long,
+      epochId: Long): DataWriter[InternalRow] = NoopWriter
+}
```

Comment:
Looks there is already .. otherwise github will show something like 
![screen shot 2019-01-24 at 2 10 45 pm](https://user-images.githubusercontent.com/6477701/51658099-df8a5b80-1fe1-11e9-81cc-7dd7ae78ac13.png)

Reply:
Yep, fixed.